Update CONTRIBUTING.md

oliver-krauss opened the pull request:
closes #106

regelkraut reviewed (commented):
The change looks good, I saw that you moved the Contributing.md content from the R4 repo to the top-level project and added a readme to make it visible. 
But this also means that we have to adapt some of the links in the new Contributing.md that still point to the R4 repo, see for example the "Support" chapter.

regelkraut commented:
Closes #106

oliver-krauss commented:
@regelkraut please see https://github.com/HL7Austria/.github/blob/main/CONTRIBUTING.md I changed the texts (2x "for example see") and removed the link at the support chapter entirely. If there are any issues left feel free to edit, or create new issues in the contributing repo.
